web/src/app/workplace/taskdetails/taskdetails.component.ts

```diff
@@ -101,7 +139,7 @@ export class TaskdetailsComponent implements OnInit, OnDestroy {
   backClicked(): void {
     this.task = undefined;
     this.taskService.selectTask(this.task);
-    this.router.navigate(['./'], { relativeTo: this.route.parent });
+    this.router.navigate(['./'], {relativeTo: this.route.parent});
```

Comment:
If you go directly to -> http://localhost:4200/workplace/tasks/(detail:taskdetail/new-task)
deeplinking doesn't work but new-task screen is been loaded, allow deepliking or avoid an user navigate directly to this route